LibSWBF2/Chunks/BaseChunk.cpp

```diff
@@ -151,15 +151,15 @@ namespace LibSWBF2::Chunks
 		return m_ChunkPosition + sizeof(ChunkHeader) + sizeof(ChunkSize);
 	}
 
-	bool BaseChunk::PositionInChunk(const size_t& CurrentPosition)
+	bool BaseChunk::PositionInChunk(const size_t& PossiblePosition)
 	{
 		size_t dataPosition = GetDataPosition();
-		return CurrentPosition >= dataPosition && CurrentPosition < dataPosition + m_Size;
+		return PossiblePosition >= dataPosition && PossiblePosition < dataPosition + GetAlignedSize();
 	}
 
 	bool BaseChunk::ThereIsAnother(FileReader& stream)
 	{
-		return stream.GetFileSize() - stream.GetPosition() >= 8 && PositionInChunk(stream.GetPosition() + 8);
+		return stream.GetFileSize() - stream.GetPosition() >= 8 && PositionInChunk(stream.GetPosition() + 7);
```

Comment:
If the file were 8 bytes long and one called this ```ThereIsAnother``` from position 0, testing ```PositionInChunk``` with 0 + 8 would return false, when it should obviously be true.